mindjunkies/accounts/tests/test_view.py

```
@@ -1,25 +1,30 @@
 import pytest
 from django.urls import reverse
-from mindjunkies.accounts.models import User
+from decouple import config
 
+from mindjunkies.accounts.models import User
```

Ruff (0.8.2)

`mindjunkies.accounts.models.User` imported but unused

Remove unused import: `mindjunkies.accounts.models.User`

(F401)